### PR TITLE
Downgrade Fabric8 to 4.9.2

### PR DESCRIPTION
**Pull request by istraka**
Revert "Update Fabric8 client to 4.10.1"
Downgrade Fabric8 to 4.9.2

Fixes https://github.com/xtf-cz/xtf/issues/361

Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- [x] Code is self-descriptive and/or documented

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
 		<!-- Depedency version properties -->
 		<version.httpclient>4.5.5</version.httpclient>
-		<version.openshift-client>4.10.1</version.openshift-client>
+		<version.openshift-client>4.9.2</version.openshift-client>
```
Comment (mnovak1) [inline pom.xml:46]: Do we have confidence that this will not break anything?

Comment (istraka) [reply pom.xml:46]: I will run level1 tests of our product openshift ts.

Comment (istraka) [reply pom.xml:46]: Level1 tests have passed.

**Comment by mnovak1** [comment]
Ok, merging

**Pull request merged**